.github/workflows/platform-deployment.yaml

```diff
@@ -281,7 +281,6 @@ jobs:
   deploy_client_purpose_updater:
     name: Client Purpose Updater
     needs: [deploy_configs]
-    if: ${{ inputs.environment == 'es1-dev' || inputs.environment == 'es1-test' || inputs.environment == 'es1-att' }}
```

Comment:
Since this change enables also Qa, do we have its configurations yet?

Reply:
I'm adding that configuration in a specific PR, meanwhile I fix this configuration.